app/models/installed_technology.rb

```diff
@@ -71,7 +71,8 @@ def technology
   def behavior_with_curve(curve_type = nil)
     behavior = self.behavior.presence || technology.behavior
 
-    return behavior if curve_type.nil? || curve_type == 'default'.freeze
+    return "default" if is_edsn_profile?
```

Comment:
@antw I changed these lines of code. Didn't add the behaviors yet for `base_load_edsn`.